agent/provider/readme.md

```diff
@@ -38,36 +27,77 @@ This can be displayed using `cargo run --bin ya-provider run --help`
 To obtain `YAGNA_APPKEY` we need to be in this newly created workdir `cd ya-prov`:
 
 1. Run [yagna service](https://github.com/golemfactory/yagna/blob/master/core/serv/README.md):
-    ```
-    cargo run --bin yagna -- service run
+    ```bash
+    cargo run service run
     ```
     If you want to set `debug` log level or higher its good to filter out core crates to `info`:
-    ```
-    RUST_LOG=debug,tokio_core=info,tokio_reactor=info,hyper=info cargo run --bin yagna -- service run
+    ```bash
+    RUST_LOG=debug,tokio_core=info,tokio_reactor=info,hyper=info cargo run service run
     ```
 
 2. Create app-key token
 
-    In another console, go to the same directory and run:
-    ```
-    cargo run --bin yagna -- app-key create "provider-agent"
-    ```
-    it will display newly created app-key eg.
-    ```
-    $ cargo run --bin yagna -- app-key create "provider-agent"
+    In another console, go to the same directory and run:\
+    (it will display newly created app-key)
+    ```bash
+    $ cargo run app-key create "provider-agent"
     58cffa9aa1e74811b223b627c7f87aac
     ```
 
 3. Put this app-key into your `.env` file as a value for variable `YAGNA_APPKEY`.
 
+## ExeUnits
+
+## WASI (wasmtime)
+
+This is the first ExeUnit we've prepared for you.
+You need to clone its repository and build.
+In following sections we assume you've cloned it to the same directory where `yagna` is cloned.
+```
+cd ../..  # assuming you are in ./yagna/ya-prov
```

Comment:
You assume to be in provider directory, but local-debug-exeunits-descriptor.json points to _ya-runtime-wasi_ in yagna directory as I see

Reply:
nope, check carefully, there is thriple times `..` going up in tree